### PR TITLE
1 and 3 dimensional convolutional operations

### PR DESCRIPTION
**Pull request by kdduha**
* adding `nn.Conv1d` and `nn.Conv2d`
* adding `nn.TransposeConv1d` and `nn.TransposeConv2d`
* adding corresponding normalization layers (`DropoutNd` and `BatchNormNd`)
* updating `builder_examples.ipynb` with new parameters

### CODE REVIEW

**Comment by kdduha** [comment]
now you can use several `convNd_out` and `conv_transposeNd_out` without loops just by adding `n_layers`
